sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusAmqpConnection.java

```diff
@@ -45,4 +45,23 @@
      */
     Mono<AmqpReceiveLink> createReceiveLink(String linkName, String entityPath, ReceiveMode receiveMode,
         boolean isSession, String transferEntityPath, MessagingEntityType entityType);
+
+    /**
+     * Creates or gets an existing receive link. The same link is returned if there is an existing receive link with the
+     * same {@code linkName}. Otherwise, a new link is created and returned.
+     *
+     * @param linkName The name of the link.
+     * @param entityPath The remote address to connect to for the message broker.
+     * @param transferEntityPath Path if the events should be transferred to another link after being received from this
+     *     link.
+     * @param receiveMode {@link ReceiveMode} to use when creating the link.
+     * @param isSession to use when creating the link.
+     * @param sessionId to use when creating the link.
+     * @param transferEntityPath to use when creating the link.
+     * @param entityType {@link MessagingEntityType} to use when creating the link.
+     *
+     * @return A new or existing receive link that is connected to the given {@code entityPath}.
+     */
+    Mono<AmqpReceiveLink> createReceiveLink(String linkName, String entityPath, ReceiveMode receiveMode,
```

Comment:
This would be an overload where you don't need "isSession" because we'd have a sessionId. And the parameter would be at the end.